my_css.css

```diff
@@ -0,0 +1,114 @@
+* {
+    margin: 0;
+    padding: 0;
+}
```

Comment:
Please do not use selector *. It is too heavy operation and it takes a lot of efforts